### PR TITLE
feat: remove CurrentBlock storage from starknet pallet

### PR DESCRIPTION
**Pull request by chirag-bgh**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Feature

## What is the current behavior?

Resolves: #531 

## What is the new behavior?

- removes CurrentBlock storage from starknet pallet

## Does this introduce a breaking change?

No

### CODE REVIEW

File: crates/primitives/starknet/src/tests/execution.rs [hunk 1]
```diff
@@ -2,17 +2,23 @@ use alloc::sync::Arc;
...
 use frame_support::{assert_ok, bounded_vec};
 use hex::FromHex;
 use starknet_api::api_core::{ChainId, ClassHash, ContractAddress, EntryPointSelector, PatriciaKey};
+<<<<<<< HEAD
 use starknet_api::deprecated_contract_class::{EntryPoint, EntryPointOffset, EntryPointType};
+use starknet_api::block::{BlockNumber, BlockTimestamp};
+=======
+use starknet_api::block::{BlockNumber, BlockTimestamp};
+use starknet_api::deprecated_contract_class::EntryPointType;
+>>>>>>> 48ba247 (feat: remove CurrentBlock storage from starknet pallet)
```
Comment (lambda-0x) [inline crates/primitives/starknet/src/tests/execution.rs:17]: looks like git merge messed this up

File: crates/client/rpc/src/madara_backend_client.rs [hunk 2]
```diff
@@ -14,6 +14,7 @@ where
     C: HeaderBackend<B> + 'static,
 {
     let substrate_hashes = backend.mapping().block_hash(&hash)?;
+    log::info!("load_hash: substrate_hashes: {:?}", substrate_hashes);
```
Comment (abdelhamidbakhta) [inline crates/client/rpc/src/madara_backend_client.rs:17]: please remove this log

**Review by abdelhamidbakhta — DISMISSED** [review]
left comment

**Review by abdelhamidbakhta — APPROVED** [review]
lgtm

**Pull request merged**